labeller/manifest.go

```diff
@@ -285,6 +285,11 @@ func garbageCollectManifests(podclient corev1.PodInterface, manifestclient secsc
 		)
 		updatedManifest, updated = removeDanglingPods(currentPodKeys, manifest)
 
+		// Noop label
+		if _, ok := updatedManifest.GetLabels()["imagemanifestvuln.example"]; ok {
+			return nil
+		}
```

Comment:
Put this before `updatedManifest` is called.

Reply:
Done